### PR TITLE
hardware milestone added

### PR DESCRIPTION
**Pull request by MadalinaEliana**
### Pull Request Overview

This pull request adds hardware milestone for the project

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Stefan Madalina-Eliana <madalina.stefan2013@gmail.com>

### CODE REVIEW

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Pleaee move the kicad files to your project's repository.

File: docs/project/2024/madalina.stefan2007/images/WhatsApp Image 2024-05-19 .jpg
Comment (alexandruradovici) [inline docs/project/2024/madalina.stefan2007/images/WhatsApp Image 2024-05-19 .jpg:1]: Rename these files

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/madalina.stefan2007/images/hardware1.jpg
Comment (alexandruradovici) [inline docs/project/2024/madalina.stefan2007/images/hardware1.jpg:1]: Have you used these files in the documentation?

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/madalina.stefan2007/images/hardware1.jpg
Comment (MadalinaEliana) [reply docs/project/2024/madalina.stefan2007/images/hardware1.jpg:1]: i haven't. i thought they were supposed to be just for you to see our projects. i'll add them now.

**Review by alexandruradovici — DISMISSED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**